cli/command/service/create.go

```diff
@@ -39,6 +39,8 @@ func newCreateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.StringSliceVar(&opts.constraints, flagConstraint, []string{}, "Placement constraints")
 	flags.StringSliceVar(&opts.networks, flagNetwork, []string{}, "Network attachments")
 	flags.VarP(&opts.endpoint.ports, flagPublish, "p", "Publish a port as a node port")
```

Comment:
Do we need all 3 of these?

Doesn't the old flag do `ingress`? So can't we keep the existing flag for ingress and add only `publish-host` ?